public_website/src/hooks/useAxeptio.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react'
+
+export const useAxeptio = () => {
+  useEffect(() => {
+    const script = document.createElement('script')
+    script.async = true
+    script.innerHTML = `
+      window.axeptioSettings = { clientId: "66bcc4aa49e3d3d814d9f71b", cookiesVersion: "siteinstit testing-fr-EU", };
+      (function(d, s) {
+        var t = d.getElementsByTagName(s)[0], e = d.createElement(s);
+        e.async = true; e.src = "//static.axept.io/sdk.js";
```

Comment:
[Dans la doc je vois qu'il faut mettre ça plutôt pour react](https://developers.axeptio.eu/sdk/integration-du-sdk#le-cas-react) :
```suggestion
        e.async = true; e.src = "//static.axept.io/sdk-slim.js";
```
Ça semble être la version light d'Axeptio (https://developers.axeptio.eu/sdk/sdk-slim)

Reply:
Je crois que tu n'as pas commit le changement du coup ? :D